frog/imports/ui/TeacherView.jsx

```diff
@@ -104,7 +104,7 @@ const SessionList = ( { sessions } ) => { return(
 )}
 
 const DashView = ({ user, logs }) => {
-  const session = user.profile? Sessions.findOne({_id:user.profile.currentSession}):null
+  const session = user.profile? Sessions.findOne({_id:user.profile.controlSession}):null
```

Comment:
:) Funny, of course I haven't seen it since you have not done a pull request yet :)